homeassistant/components/deconz/light.py

```diff
@@ -137,21 +163,20 @@ def effect_list(self):
     @property
     def color_temp(self):
         """Return the CT color value."""
-        if self._device.colormode != "ct":
-            return None
-
         return self._device.ct
 
     @property
     def hs_color(self):
         """Return the hs color value."""
-        if self._device.colormode in ("xy", "hs"):
-            if self._device.xy:
-                return color_util.color_xy_to_hs(*self._device.xy)
-            if self._device.hue and self._device.sat:
-                return (self._device.hue / 65535 * 360, self._device.sat / 255 * 100)
+        if self._device.hue is not None and self._device.sat is not None:
```

Comment:
The base component will only read the `hs_color` attribute if the light reports `color_mode=hs`, so you may not need this check.

Reply:
Something broke when I didn't have this control the other day because that was my assumption as well. I will remote it and see whats breaking

Reply:
Right now its needed for groups, because how its handled in the library. I will review how to improve the library to remove this all together.

Fixed by overriding solution in group class